### PR TITLE
Check mahuika before submission

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

File: scripts/cybershake/auto_submit.py [hunk 1]
```diff
@@ -246,7 +249,7 @@ def get_vmname(srf_name):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("run_folder", type=str, help="folder to the collection of runs")
-    parser.add_argument("--n_runs", default=default_n_runs, type=int)
+    parser.add_argument("--n_runs", default=None, type=list, nargs="+")
```
Comment (jasonmotha) [inline scripts/cybershake/auto_submit.py]: Add help txt

Comment (claudio525) [inline scripts/cybershake/auto_submit.py]: Maye add a help text for this?

File: scripts/cybershake/auto_submit.py [hunk 2]
```diff
@@ -256,22 +259,36 @@ def main():
...
-    parser.add_argument("--no_tidy_up", action="store_true")
+    parser.add_argument("--no_clean_up", action="store_true")
 
     args = parser.parse_args()
 
-    n_runs_max = args.n_runs
     mgmt_db_location = args.run_folder
     db = connect_db(mgmt_db_location)
-    tidy_up = not args.no_tidy_up
+    # clean_up = not args.no_clean_up
```
Comment (claudio525) [inline scripts/cybershake/auto_submit.py]: Remove it if its no longer needed?

File: scripts/cybershake/auto_submit.py [hunk 3]
```diff
@@ -249,7 +249,14 @@ def get_vmname(srf_name):
...
     parser = argparse.ArgumentParser()
     parser.add_argument("run_folder", type=str, help="folder to the collection of runs")
-    parser.add_argument("--n_runs", default=None, type=list, nargs="+")
+    parser.add_argument(
+        "--n_runs",
+        default=None,
+        type=list,
+        nargs="+",
+        help="The number of processes each machine can run at once. If a single value is given this is used for all "
+             "machines, otherwise one value per machine must be given.",
```
Comment (jasonmotha) [inline scripts/cybershake/auto_submit.py]: State the order of which machine is first. (can be populated automagically from the enum/list/object)

**Pull request merged**